backend/apps/challenges/views.py

```
@@ -150,14 +149,15 @@ class BulkChallengeUploadView(APIView):
     """
 
     def get_permissions(self):
-        from apps.rbac.permissions import HasPermission
         from rest_framework import permissions
 
+        from apps.rbac.permissions import HasPermission
+
         return [permissions.IsAuthenticated(), HasPermission("create_content")]
 
     parser_classes = [MultiPartParser]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)